python/cuml/test/test_fil.py

```diff
@@ -17,27 +17,25 @@
 import pytest
 import os
 
-from cuml import FIL, fil
-from cuml.utils.import_utils import has_treelite, has_xgboost
+from cuml import FIL
+from cuml.utils.import_utils import has_xgboost
 
-from sklearn.datasets import make_classification
-from sklearn.metrics import accuracy_score
-
-if has_treelite():
-    import treelite as tl
+from sklearn.datasets import make_classification, make_regression
+from sklearn.metrics import accuracy_score, mean_squared_error
 
 if has_xgboost():
     import xgboost as xgb
 
-
     def simulate_data(m, n, k=2, random_state=None, classification=True):
         if classification:
             features, labels = make_classification(n_samples=m, n_features=n,
-                                                   n_informative=int(n/5), n_classes=k,
-                                                  random_state=random_state)
+                                                   n_informative=int(n/5),
+                                                   n_classes=k,
+                                                   random_state=random_state)
         else:
             features, labels = make_regression(n_samples=m, n_features=n,
-                                               n_informative=int(n/5), n_targets=1,
+                                               n_informative=int(n/5),
+                                               n_targets=1,
                                                random_state=random_state)
         return np.c_[labels, features].astype(np.float32)
```

Comment:
Better not to concatenate here. You're just going to split back to (labels, features) in the future. Keeping them split as a tuple would match the sklearn.datasets apis